greet.js

```diff
@@ -0,0 +1,10 @@
+export function greet(name){
+    const names = ['Thomas','Stefan','Martin'];
+    if (names.includes(name)){
+        return 'Hello Coach!';
+    }else if (name != null){
+        return 'Hello ' + name + '!';
+    }else {
+        return 'Hello stranger!';
+    }
```

Comment:
You don't need the `else if` here. Just the `if` would be enough because in the first `if` you're using the `return` statement. So either the if is truthy and the functions returns here or the code continues anyway.
The last `else` is redundant then, because if everythings fails, just return `Hello stranger` 

```suggestion
    if (names.includes(name)) {
    return 'Hello Coach!'
  }
  if (name != null) {
    return 'Hello ' + name + '!'
  }
  return 'Hello stranger!'
}
```